docs/release-procedure.md

```diff
@@ -3,7 +3,8 @@
 ### Point release
```

Comment:
```suggestion
### Patch release
```
Since we use the semantic versioning I would prefer we move to the original terminology in order not to confuse users and us ourselves. The "Major release" section below should probably be changed to "Minor and Major releases".

https://semver.org/

Reply:
Makes sense!